src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -1,9 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="Xamarin.Android.Tools.BootstrapTasks.GenerateProfile" AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" />
+  <UsingTask TaskName="Xamarin.Android.Tools.BootstrapTasks.UnzipDirectoryChildren" AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" />
   <PropertyGroup>
     <_SharedRuntimeBuildPath Condition=" '$(_SharedRuntimeBuildPath)' == '' ">..\..\bin\$(Configuration)\lib\xbuild-frameworks\MonoAndroid\</_SharedRuntimeBuildPath>
     <_GeneratedProfileClass>$(IntermediateOutputPath)Profile.g.cs</_GeneratedProfileClass>
+    <BuildDependsOn>
+		$(BuildDependsOn);
+		_CopyExtractedMultiDexJar;
+    </BuildDependsOn>
+    <_AndroidSdkLocation>$(ANDROID_SDK_PATH)</_AndroidSdkLocation>
```

Comment:
> Want me to eliminate the variables that are used ONLY in monodroid in the same manner? For example, InstantRunEnabled.

If they *can* be removed, that would likely be best. If they *can't* be removed -- which may be the case for `$(_InstantRunEnabled)`, which is used with the `<DetermineJavaLibrariesToCompile/>` task -- then perhaps it should be renamed to be something more meaningful w/o context/memory/history.

For example, instead of `DetermineJavaLibrariesToCompile.EnableInstantRun`, rename it to `DetermineJavaLibrariesToCompile.SkipMonoPlatformJar`. For those just reading `Xamarin.Android.Common.targets`, I assume that seeing `SkipMonoPlatformJar="$(_SkipMonoPlatformJar)"` would be more meaningful than `EnableInstantRun="$(_InstantRunEnabled)"`.